lib/cursor.js

```diff
@@ -143,8 +143,10 @@ function Cursor(bson, ns, cmd, options, topology, topologyOptions) {
     promiseLibrary: promiseLibrary,
     // Current doc
     currentDoc: null,
+    // expicitlyDoNotUseSession
+    expicitlyDoNotUseSession: options.expicitlyDoNotUseSession,
```

Comment:
we already have a `skipSessions` elsewhere in the code, would it make sense to keep naming consistent even though they are in disparate locations?

Reply:
I kind of like @kvwalker's `explicitlyIgnoreSession`. I want to make sure we do not confuse the `skipSession` variable with this.

Reply:
SGTM